src/script/backport-resolve-issue

```diff
@@ -352,8 +352,9 @@ def tag_sha1(repo, sha1):
 def ver_to_release():
     return {'v9.2': 'infernalis', 'v10.2': 'jewel', 'v11.2': 'kraken',
             'v12.2': 'luminous', 'v13.2': 'mimic', 'v14.2': 'nautilus',
-            'v15.2': 'octopus', 'v16.0': 'pacific', 'v16.1': 'pacific',
-            'v16.2': 'pacific', 'v17.0': 'quincy', 'v18.0': 'reef'}
+            'v15.2': 'octopus', 'v16.0': 'pacific', 'v16.1': 'pacific', 
+            'v16.2': 'pacific', 'v17.0': 'quincy', 'v17.2': 'quincy',
+            'v18.0': 'reef', 'v18.1': 'reef'}
```

Comment:
We recently released 18.2.0, so you can also add that to the script, like this:
```suggestion
            'v18.0': 'reef', 'v18.1': 'reef', 'v18.2': reef'}
```

But that can be done in a later PR.